lib/logstash/inputs/sqs.rb

```diff
@@ -83,6 +83,9 @@ class LogStash::Inputs::SQS < LogStash::Inputs::Threadable
   # Name of the SQS Queue name to pull messages from. Note that this is just the name of the queue, not the URL or ARN.
   config :queue, :validate => :string, :required => true
 
+  # Account ID of the AWS account which owns the queue.
+  config :queue_aws_account_id, :validate => :string, :required => false
+
```

Comment:
We will also need to add new user documentation in the static asciidoc file at https://github.com/logstash-plugins/logstash-input-sqs/blob/master/docs/index.asciidoc